data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -8,22 +8,32 @@
                 <author>Hanno</author>
                 <editor xml:lang="lat">Carolus Müllerus</editor>
                 <sponsor>Harvard College Library</sponsor>
```

Comment:
Also delete the `<sponsor>` tag, since this information is given instead in the `<funder>` element.

Reply:
Do you mean to delete the <author> and <editor> tags?